NAMESPACE

```diff
@@ -18,17 +19,25 @@ export(summarize_portfolio_production)
 export(summarize_weighted_production)
 export(sym)
 export(syms)
+importFrom(dplyr,arrange)
```

Comment:
Imported functions from dplyr to reduce "noise" so I can see what matters more clearly.